vizro-core/src/vizro/models/types.py

```diff
@@ -239,8 +242,8 @@ class capture:
 
     """
 
-    def __init__(self, mode: Literal["graph", "action", "table"]):
-        """Instantiates the decorator to capture a function call. Valid modes are "graph", "table" and "action"."""
+    def __init__(self, mode: Literal["graph", "action", "table", "aggrid"]):
```

Comment:
```suggestion
    def __init__(self, mode: Literal["graph", "action", "table", "ag_grid"]):
```

I think we should have a `_` here (change applies in many places).